projects/paintswap/index.js

```diff
@@ -1,14 +1,61 @@
-const {calculateUsdUniTvl} = require('../helper/getUsdUniTvl.js')
+const web3 = require("../config/web3.js");
```

Comment:
Doesn't this use an ETH RPC endpoint?

Reply:
It uses what ever the environment variable is set to. I guess if it's always points to an Ethereum RPC server on your end, then that would be a problem

Reply:
I am using the acclaimed rpc.ftm.tools endpoint now. Was recently acquired by ANKR https://twitter.com/ankr/status/1439529700192292864